setup.py

```diff
@@ -16,13 +14,12 @@
     license="BSD (3-clause)",
     url="https://github.com/NSLS-II/bluesky",
     packages=setuptools.find_packages(),
-    package_data={'bluesky': ['schema/*.json']},
     scripts=glob.glob('scripts/*'),
     install_requires=requirements,
     classifiers=[
         "Development Status :: 3 - Alpha",
-        "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```

Comment:
Should we also add changes to PyPI description page, same as https://github.com/NSLS-II/ophyd/pull/583/files#r198996869?